liquibase-couchbase/src/main/java/liquibase/ext/couchbase/validator/MutateInValidator.java

```diff
@@ -42,8 +42,8 @@ protected boolean isValueEmpty(Value value) {
             .isPresent();
     }
 
-    protected void validateNoMultipleValues(List<Value> values) {
-        if (!values.isEmpty()) {
+    protected void validateMultipleValuesPresence(List<Value> values) {
+        if (values.isEmpty()) {
             throw new MutateInMultipleValuesNotAllowedException(mutateInType);
         }
```

Comment:
If values is empty we throw `MutateInMultipleValuesNotAllowedException` ? Please look at the comments above where I explained why it's wrong